### PR TITLE
Format ballerina doc and error messages

### PR DESCRIPTION
**Pull request by MohamedSabthar**

### CODE REVIEW

File: udp-ballerina/connectionless_client.bal [hunk 1]
```diff
@@ -16,14 +16,16 @@
...
-# Initializes the UDP connectionless client based on the provided configurations.
+# Initializes the UDP connectionless client based on the provided configurations
 #
-# + localHost - The local IP address string in textual presentation to which the socket is bound
+# + localHost - The local IP address string in textual presentation to which
+#                the socket is bound
 public client class Client {
  
-    private string? localHost = ();
+    // private string? localHost = ();
```
Comment (Maninda) [inline udp-ballerina/connectionless_client.bal:25]: If not required shall we remove this line?

File: udp-native/src/main/java/org/ballerinalang/stdlib/udp/endpoint/ConnectionlessClientActions.java [hunk 2]
```diff
@@ -80,18 +80,18 @@ public static Object initEndpoint(Environment env, BObject client, BMap<BString,
             selectorManager.start();
         } catch (SelectorInitializeException e) {
             log.error(e.getMessage(), e);
-            balFuture.complete(SocketUtils.createSocketError("unable to initialize the selector"));
+            balFuture.complete(SocketUtils.createSocketError("Unable to initialize the selector."));
             return null;
         } catch (SocketException e) {
-            balFuture.complete(SocketUtils.createSocketError("unable to bind the local udp port"));
+            balFuture.complete(SocketUtils.createSocketError("Unable to a port."));
```
Comment (Maninda) [inline udp-native/src/main/java/org/ballerinalang/stdlib/udp/endpoint/ConnectionlessClientActions.java:86]: "Unable to a port." - Shall we make make this more expressive?

File: udp-native/src/main/java/org/ballerinalang/stdlib/udp/endpoint/ConnectionlessClientActions.java [hunk 3]
```diff
@@ -129,7 +129,7 @@ public static Object send(BObject client, BMap<BString, Object> datagram) {
             }
             return null;
         } catch (ClosedChannelException e) {
-            return SocketUtils.createSocketError("client udp close already.");
+            return SocketUtils.createSocketError("udp client closed already.");
```
Comment (Maninda) [inline udp-native/src/main/java/org/ballerinalang/stdlib/udp/endpoint/ConnectionlessClientActions.java:132]:
```suggestion
            return SocketUtils.createSocketError("UDP client is already closed.");
```

File: udp-ballerina/connectionless_client.bal (hunk 1 above)
Comment (MohamedSabthar) [reply udp-ballerina/connectionless_client.bal:25]: removed

File: udp-native/src/main/java/org/ballerinalang/stdlib/udp/endpoint/ConnectionlessClientActions.java (hunk 2 above)
Comment (MohamedSabthar) [reply udp-native/src/main/java/org/ballerinalang/stdlib/udp/endpoint/ConnectionlessClientActions.java:86]: fixed

File: udp-native/src/main/java/org/ballerinalang/stdlib/udp/endpoint/ConnectionlessClientActions.java (hunk 3 above)
Comment (MohamedSabthar) [reply udp-native/src/main/java/org/ballerinalang/stdlib/udp/endpoint/ConnectionlessClientActions.java:132]: updated

**Review by Maninda — APPROVED** [review]

**Pull request merged**